training/src/main/java/com/wolox/training/models/Book.java

```diff
@@ -0,0 +1,45 @@
+package com.wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+public class Book {
+
+  @Id
```

Comment:
If we don't set a GeneratedValue strategy it will take the AUTO strategy as the default. This strategy uses a global auto increment in the case of PostgreSQL, which is not the best choice for this case
So, knowing that we'll be using PostgreSQL, we should use the GeneratedValue Sequence strategy. So we can have a sequence per table (totally independent from the others)